drivers/SmartThings/matter-sensor/src/air-quality-sensor/init.lua

```diff
@@ -46,41 +46,71 @@ local subscribed_attributes = {
   },
+  [capabilities.carbonMonoxideHealthConcern.ID] = {
+    clusters.CarbonMonoxideConcentrationMeasurement.attributes.LevelValue,
+  },
   [capabilities.carbonDioxideMeasurement.ID] = {
     clusters.CarbonDioxideConcentrationMeasurement.attributes.MeasuredValue,
     clusters.CarbonDioxideConcentrationMeasurement.attributes.MeasurementUnit,
   },
+  [capabilities.carbonDioxideHealthConcern.ID] = {
+    clusters.CarbonDioxideConcentrationMeasurement.attributes.LevelValue,
+  },
   [capabilities.nitrogenDioxideMeasurement.ID] = {
     clusters.NitrogenDioxideConcentrationMeasurement.attributes.MeasuredValue,
     clusters.NitrogenDioxideConcentrationMeasurement.attributes.MeasurementUnit
   },
+  [capabilities.nitrogenDioxideHealthConcern.ID] = {
+    clusters.NitrogenDioxideConcentrationMeasurement.attributes.LevelValue,
+  },
   [capabilities.ozoneMeasurement.ID] = {
     clusters.OzoneConcentrationMeasurement.attributes.MeasuredValue,
     clusters.OzoneConcentrationMeasurement.attributes.MeasurementUnit
   },
+  [capabilities.ozoneHealthConcern.ID] = {
+    clusters.OzoneConcentrationMeasurement.attributes.LevelValue,
+  },
   [capabilities.formaldehydeMeasurement.ID] = {
     clusters.FormaldehydeConcentrationMeasurement.attributes.MeasuredValue,
     clusters.FormaldehydeConcentrationMeasurement.attributes.MeasurementUnit,
   },
+  [capabilities.formaldehydeHealthConcern.ID] = {
+    clusters.FormaldehydeConcentrationMeasurement.attributes.LevelValue,
+  },
   [capabilities.veryFineDustSensor.ID] = {
     clusters.Pm1ConcentrationMeasurement.attributes.MeasuredValue,
     clusters.Pm1ConcentrationMeasurement.attributes.MeasurementUnit,
   },
+  [capabilities.veryFineDustHealthConcern.ID] = {
+    clusters.Pm1ConcentrationMeasurement.attributes.LevelValue,
+  },
   [capabilities.fineDustSensor.ID] = {
     clusters.Pm25ConcentrationMeasurement.attributes.MeasuredValue,
     clusters.Pm25ConcentrationMeasurement.attributes.MeasurementUnit,
   },
+  [capabilities.fineDustHealthConcern.ID] = {
+    clusters.Pm25ConcentrationMeasurement.attributes.LevelValue,
+  },
   [capabilities.dustSensor.ID] = {
     clusters.Pm10ConcentrationMeasurement.attributes.MeasuredValue,
     clusters.Pm10ConcentrationMeasurement.attributes.MeasurementUnit,
   },
+  [capabilities.dustHealthConcern.ID] = {
+    clusters.Pm10ConcentrationMeasurement.attributes.LevelValue,
+  },
   [capabilities.radonMeasurement.ID] = {
     clusters.RadonConcentrationMeasurement.attributes.MeasuredValue,
     clusters.RadonConcentrationMeasurement.attributes.MeasurementUnit,
   },
+  [capabilities.radonHealthConcern.ID] = {
+    clusters.RadonConcentrationMeasurement.attributes.LevelValue,
+  },
   [capabilities.tvocMeasurement.ID] = {
     clusters.TotalVolatileOrganicCompoundsConcentrationMeasurement.attributes.MeasuredValue,
     clusters.TotalVolatileOrganicCompoundsConcentrationMeasurement.attributes.MeasurementUnit,
+  },
+  [capabilities.tvocHealthConcern.ID] = {
```

Comment:
It'd be good to add a test case for this and the other health concern capabilities to the existing unit test case.

Reply:
Added in 2f6d6be9343a806c29c8f9bfaa5122623aa0d116.